src/diffusers/pipelines/animemory/modeling_text_encoder.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2024 AniMemory Team and The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import torch
+from safetensors.torch import load_file
+from transformers import CLIPTextConfig, CLIPTextModelWithProjection
+from transformers.models.t5.configuration_t5 import T5Config
+from transformers.models.t5.modeling_t5 import T5Stack
+
+
+class AniMemoryT5(torch.nn.Module):
```

Comment:
Would `AniMemoryT5` and `AniMemoryAltCLip` be better added to `transformers` directly?

Reply:
This two model are mostly similar to the original T5 and AltCLip, which just features some little arch modifications and tokenizer replacement. It's very unique for this model design for bilingual alignments.